CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.5.0-beta 
+
+This code is considered production quality, but depends on angular2: 
+^3.0.0-alpha+1. The alpha tag represents the evolving nature of the AngularDart 
+api, not code quality (3.0.0-beta+1 is used in production Google apps). 
+ 
+ * Rename library to angular_components. 
```

Comment:
Shoot. Messed up.

The rename should be at the TOP of the changelog. With **BOLD** text.